cli/src/command/account.rs

```diff
@@ -920,81 +967,118 @@ pub async fn voting_output_command(account: &Account) -> Result<(), Error> {
-                        .output
-                        .unlock_conditions()
-                        .expect("output must have unlock conditions");
-                    let sdr_amount = unlock_conditions
-                        .storage_deposit_return()
-                        .map(|sdr| sdr.amount())
-                        .unwrap_or(0);
-
-                    amount += output_data.output.amount() - sdr_amount;
-                }
+    for (output_id, output_data) in unspent_outputs.into_iter().map(|data| (data.output_id, data)) {
+        outputs.push((output_id, output_data.output.kind_str().to_string()));
+
+        // Output might be associated with the address, but can't be unlocked by it, so we check that here.
+        // Panic: cannot fail for outputs belonging to an account.
+        let (required_address, _) = output_data
+            .output
+            .required_and_unlocked_address(current_time, &output_id, None)
+            .unwrap();
+
+        if address.inner() == &required_address {
+            if let Some(nts) = output_data.output.native_tokens() {
+                native_tokens.add_native_tokens(nts.clone())?;
+            }
+            match &output_data.output {
+                Output::Nft(nft) => nfts.push((
+                    nft.nft_id_non_null(&output_id),
+                    nft.nft_address(&output_id).to_bech32(*address.hrp()),
+                )),
+                Output::Alias(alias) => aliases.push((
+                    alias.alias_id_non_null(&output_id),
+                    alias.alias_address(&output_id).to_bech32(*address.hrp()),
+                )),
+                Output::Basic(_) | Output::Foundry(_) | Output::Treasury(_) => {}
             }
+            let unlock_conditions = output_data
+                .output
+                .unlock_conditions()
+                .expect("output must have unlock conditions");
+            let sdr_amount = unlock_conditions
+                .storage_deposit_return()
+                .map(|sdr| sdr.amount())
+                .unwrap_or(0);
+
+            amount += output_data.output.amount() - sdr_amount;
         }
     }
 
-    log = format!(
-        "{log}\nOutputs: {:#?}\nBase coin amount: {}\nNative Tokens: {:#?}\nNFTs: {:#?}\nAliases: {:#?}\nFoundries: {:#?}\n",
-        output_ids,
-        amount,
-        native_tokens.finish_vec()?,
-        nfts,
-        aliases,
-        foundries,
-    );
+    let mut row_count = 0;
+
+    // outputs table
+    formatted_table.push_str("\nOutputs:");
+    for (id, kind) in outputs {
+        formatted_table.push_str(&format!("{0}{1:<6}{id}{0}{2:<6}{kind}", "\n  ", "Id:", "Type:"));
+        row_count += 1;
+    }
+    formatted_table.push_str(if row_count == 0 { "\n  None\n" } else { "\n" });
+
+    // base token table
+    formatted_table.push_str("\nBase Tokens:");
+    formatted_table.push_str(&format!("{0}{1:<8}{2}\n", "\n  ", "Amount:", amount));
+
+    formatted_table.push_str("\nNative Tokens:");
```

Comment:
Do you think we could not print the titles if empty ?

Reply:
yes, if you prefer that we can not print them ... currently it would print `None`.

Reply:
I tried it and it doesn't print `None`.  But I think it would be better without anyway.

Reply:
Done